packages/tpetra/core/src/Tpetra_MultiVector_def.hpp

```diff
@@ -1362,9 +1362,25 @@ namespace Tpetra {
     // Resizing currently requires calling the constructor, which
     // clears out the 'modified' flags.
     if (packOnHost) {
+      // nde 06 Feb 2020: If 'exports' does not require resize
```

Comment:
We could just call `clear_sync_flags` on `exports`, since we don't care about the DualView's contents on device.  This is a good use case for write-only access.

Reply:
@mhoemmen just to be clear, the suggested change is to replace:
```c++
      if (exports.need_sync_host ())
        exports.sync_host ();
```
with
```c++
      exports.clear_sync_flags ();
```
?

Reply:
@mhoemmen addressed in recent commit

Reply:
Thanks to @brian-kelley for sanity checking the change for me